packages/trip-details/src/index.tsx

```diff
@@ -0,0 +1,232 @@
+import coreUtils from "@opentripplanner/core-utils";
+import moment from "moment";
+import React, { ReactElement } from "react";
+import { FormattedMessage, FormattedNumber } from "react-intl";
+import { CalendarAlt, Heartbeat, MoneyBillAlt } from "styled-icons/fa-solid";
+
+import * as Styled from "./styled";
+import TripDetail from "./trip-detail";
+import { CaloriesDetailsProps, TripDetailsProps } from "./types";
+
+// Load the default messages.
+import defaultMessages from "../i18n/en-US.yml";
+
+const MESSAGE_ID_PREFIX = "otpUi.TripDetails.";
+const messageIds = {
+  calories: `${MESSAGE_ID_PREFIX}calories`,
+  caloriesDescription: `${MESSAGE_ID_PREFIX}caloriesDescription`,
+  departure: `${MESSAGE_ID_PREFIX}departure`,
+  title: `${MESSAGE_ID_PREFIX}title`,
+  tncFare: `${MESSAGE_ID_PREFIX}tncFare`,
+  transitFare: `${MESSAGE_ID_PREFIX}transitFare`
+};
+
+/**
+ * Format text bold (used with FormattedMessage).
+ */
+// TODO: Find a better place for this utility.
+function BoldText(contents) {
+  return <b>{contents}</b>;
+}
+
+/**
+ * Default rendering if no component is provided for the CaloriesDetails
+ * slot in the TripDetails component.
+ */
+function DefaultCaloriesDetails({
+  bikeSeconds,
+  calories,
+  walkSeconds
+}: CaloriesDetailsProps): React.Element {
+  return (
+    <FormattedMessage
+      defaultMessage={defaultMessages[messageIds.caloriesDescription]}
+      id={messageIds.caloriesDescription}
+      values={{
+        // eslint-disable-next-line react/display-name
+        a: contents => (
```

Comment:
In order to show that more than just an `a` tag is used here, I think it'd be good to rename this to something like `dietaryLink`.

Reply:
Updated in https://github.com/opentripplanner/otp-ui/pull/281/commits/8940535c0c10fc9e77c5aaac8b44e350cf04eff9.